2009-flu-uk.qmd

```diff
@@ -1 +1,437 @@
+     xlab = "Days since start of epidemics",
+     ylab = "Official cases count",
+     main = "Cases over Time with Holiday Periods",
+     ylim = c(0, max(data$Cases) * 1.3), # Add padding to the y-axis
+     xlim = c(min(data$Day), max(data$Day)))
+
+# Add the holiday periods
+hol_t <- c(0, 38, 85, 140, 147) # Holiday timings
+hol_v <- c(1, 0.70, 1, 0.85, 1) # Contact reduction
+
+# Summer holidays
+rect(xleft = hol_t[2], ybottom = 0, xright = hol_t[3], 
+     ytop = max(data$Cases) * 1.2,
+     col = "#87CEEB77", border = NA)
+     #col = adjustcolor("skyblue", alpha.f = 0.3), border = NA)
+text(x = mean(hol_t[2:3]), y = max(data$Cases) * 1.25, 
+     labels = "Summer Holiday", col = "blue", cex = 0.8)
+
+# Half-term holiday
+rect(xleft = hol_t[4], ybottom = 0, xright = hol_t[5], 
+     ytop = max(data$Cases) * 1.2, 
+     col = "#90EE9077", border = NA)
+text(x = mean(hol_t[4:5]), y = max(data$Cases) * 1.25, 
+     labels = "Half-Term", col = "darkgreen", cex = 0.8)
+
+# Legend
+legend("topright", legend = c("Cases", "Summer Holiday", "Half-Term"),
+       pch = c(19, NA, NA), col = c("red", "#87CEEB77", "#90EE9077"),
+       lty = c(NA, 1, 1), lwd = c(NA, 10, 10), 
+       bg = "white", inset = 0.02, bty = "n")
+
+```
+
+
+## Model design
+
+### Transmission model
+
+Similarly to the SIR model, we saw already, the SEIR model divides the population into four compartments:
+
+- **S**usceptible: Individuals at risk of infection.  
+- **E**xposed: Infected individuals who are not yet infectious.  
+- **I**nfectious: Actively transmitting the disease.  
+- **R**ecovered: Immune individuals.  
+
+Additionally, the model incorporates time-varying transmission rates to account for changes in contact patterns during holidays.
+
+The SEIR model is governed by the following system of ordinary differential equations:
+
+$$
+\left\{
+\begin{aligned}
+    \frac{dS}{dt} &= - \beta(t) \frac{S I}{N} \\
+    \frac{dE}{dt} &= \beta(t) \frac{S I}{N} - \gamma E \\
+    \frac{dI}{dt} &= \gamma - \sigma I \\
+    \frac{dR}{dt} &= \sigma I
+\end{aligned}
+\right.
+$$
+
+where:
+
+- $\beta(t)$ is the time-dependent transmission rate, representing the rate at which susceptible individuals become exposed through contact with infectious individuals.
+- $\gamma$ is the rate at which exposed individuals transition to the infectious stage (the inverse of the incubation period).
+- $\sigma$ is the recovery rate, indicating the rate at which infectious individuals recover or are removed from the infectious population.
+- $N$ is the total population size, assumed to remain constant.
+- $S(t)$, $E(t)$, $I(t)$, and $R(t)$ represent the number of susceptible, exposed, infectious, and recovered individuals, respectively, at time $t$. 
+
+We assume that the contact rate is constant outside of the holiday period, and that it drops by 30% during the summer and 15% during the half-term. We have thus:
+
+$$\beta(t) = R_{0}*\sigma * h(t)$$
```

Comment:
maybe worth explaining why sigma (or gamma if you switch names) appears here?

Reply:
we can also change earlier sections to use sigma, but it should be consistent across the book